src/main/java/formflow/library/ScreenController.java

```diff
@@ -55,10 +57,12 @@
 @EnableAutoConfiguration
 @Slf4j
 @RequestMapping(ScreenController.FLOW)
+@Validated
 public class ScreenController extends FormFlowController {
 
   public static final String FLOW = "/flow";
   public static final String FLOW_SCREEN_PATH = "{flow}/{screen}";
+  private final String UUID_REGEX = "[0-9a-fA-F]{8}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{12}";
```

Comment:
Move this to regex util file. 
Use this on other  spots where uuid is passed in as well, in this file.